SingularityUI/app/templates/tasksTable/tasksActiveBody.hbs

```diff
@@ -4,15 +4,15 @@
             <thead>
                 <tr>
                     <th data-sort-attribute="taskId.id">
-                        Name
+                        Task ID
                     </th>
                     <th class="hidden-xs" data-sort-attribute="taskId.startedAt">
-                        Started
+                        Started At
                     </th>
                     <th class="hidden-xs" data-sort-attribute="host">
                         Host
                     </th>
-                    <th class="hidden-xs" data-sort-attribute="taskId.rackId">
+                    <th class="visible-lg" data-sort-attribute="taskId.rackId">
```

Comment:
>Also, the actions icons (kill task and JSON) can fit without causing a change in the way the rest of the table looks even at the smallest sizes.
This does not appear to be the case, however, on the other tasks tables. Only on active. So I'm going to remove hidden-xs for the actions icons for active tasks only.